README.md

```diff
@@ -27,48 +34,55 @@ brew install quarkusio/tap/quarkus
 
 ## Running the server in dev mode
 
-First, launch the authentication service before running the server (see documentation [here](kubernetes/local/README.md)).
+First, launch the authentication service before running the server (see
+documentation [here](kubernetes/local/README.md)).
```

Comment:
```suggestion
documentation [here](https://github.com/DARPA-ASKEM/orchestration)).
```

Reply:
fixed

Reply:
In the future, for this kind of correction, do you guys prefer that I submit a commit directly?
When it's just obvious simple information, like documentation, etc.